pkg/resourceinterpreter/defaultinterpreter/aggregatestatus_test.go

```diff
@@ -534,6 +600,25 @@ func TestAggregatePodStatus(t *testing.T) {
 			aggregatedStatusItems: aggregatedStatusItems2,
 			expectedObj:           newObjFailed,
 		},
+		// More details please refer to: https://github.com/karmada-io/karmada/issues/2137
+		{
+			name:                  "failed pod status",
+			curObj:                curObj,
+			aggregatedStatusItems: aggregatedStatusItemsFail, //  Running + Failed => Failed
+			expectedObj:           failObj,
+		},
+		{
+			name:                  "succeeded pod status",
+			curObj:                curObj,
+			aggregatedStatusItems: aggregatedStatusItemsSucceeded, //  Running + Succeeded => Running
+			expectedObj:           succeededObj,
+		},
+		{
+			name:                  "succeeded pod status",
+			curObj:                curObj,
+			aggregatedStatusItems: aggregatedStatusItemsPending, //  Running + nil => Pending
+			expectedObj:           pendingObj,
```

Comment:
The test `name` (succeeded pod status) is not correct.

Reply:
Done!